ocaml/xenops/device.ml

```diff
@@ -1775,27 +1775,31 @@ let cmdline_of_disp info =
 	in
 	let videoram_opt = ["-videoram"; string_of_int info.video_mib] in
 	let dom0_input_opts x =
-	  (maybe_with_default [] (fun i -> ["-dom0-input"; string_of_int i]) x)
+		(maybe_with_default [] (fun i -> ["-dom0-input"; string_of_int i]) x)
+	in
+	let vnc_opts_of ip_addr_opt auto port keymap =
```

Comment:
The naming of values is a bit inconsistent: the IP address and the keymap are both optional values but one has `opt` in its name and the other doesn't. 

Independent of that I've tried to come up with more regular code but haven't found something that is vastly better.